nix/pkgs/gomobile/default.nix

```diff
@@ -2,12 +2,12 @@
   stdenv,
   lib,
   fetchgit,
-  fetchurl,
   buildGo122Module,
   zlib,
   makeWrapper,
   xcodeenv,
   androidenv,
+  darwin,
```

Comment:
You don't even use this, why?

Reply:
I only placed it here for when I was overriding `darwin` from `overlays.nix`
I reverted the changes over there so I don't need this now.